### PR TITLE
Migrate LaserScanDisplay

### LINKED ISSUE #79: Migrate LaserScanDisplay

### PR DESCRIPTION
**Pull request by greimela-si**
Now that https://github.com/ros-perception/laser_geometry/pull/28 has been merged we can migrate the LaserScanDispay to RViz2.

**Requirement: `ros/angles:ros2` and `ros-perception/laser_geometry:ros2` must be added to [ros2.repos](https://github.com/ros2/ros2/blob/master/ros2.repos) before merging.**

Resolves #79.

### CODE REVIEW

**Comment by greimela-si** [comment]
CI including the two additional repositories:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4153/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1234/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3441/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4258/)

**Comment by wjwwood** [comment]
I resolved the conflicts in this pr and made a pr for adding the new repositories: https://github.com/ros2/ros2/pull/477

Here's a new CI:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4163/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1242/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3454/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4267/)

File: rviz_default_plugins/CMakeLists.txt [hunk 1]
```diff
@@ -117,9 +120,11 @@ target_link_libraries(rviz_default_plugins
   rviz_rendering::rviz_rendering
   rviz_common::rviz_common
   resource_retriever::resource_retriever
+  laser_geometry::laser_geometry
```
Comment (wjwwood) [inline rviz_default_plugins/CMakeLists.txt]: nitpick: alphabetical order unless the library order matters (I don't think it does)

File: rviz_default_plugins/package.xml [hunk 2]
```diff
@@ -32,6 +32,7 @@
   <depend>rviz_rendering</depend>
   <depend>urdf</depend>
   <depend>visualization_msgs</depend>
+  <depend>laser_geometry</depend>
```
Comment (wjwwood) [inline rviz_default_plugins/package.xml]: nitpick: alphabetical order

**Review by wjwwood — APPROVED** [review]
two nitpicks, which I might not get to this afternoon, but otherwise lgtm

**Pull request merged**